justfile

```diff
@@ -1,3 +1,5 @@
+# this is a comment
```

Comment:
This is not completely necessary, but you didn't document what a comment was anywhere so I thought it would be helpful.